js/ui/lookup.js

```diff
@@ -61,6 +62,16 @@ const Lookup = DropDownList.inherit({
     },
 
     _getDefaultOptions: function() {
+        const getSize = (side) => {
+            let size;
+            if(devices.real().deviceType === 'phone' && window.visualViewport) {
+                size = window.visualViewport[side];
+            } else {
+                size = side === 'width' ? getWidth(window) : getHeight(window);
+            }
+            return size * WINDOW_RATIO;
+        };
```

Comment:
This function is not specific for lookup => it should not be places here. I think we should update core utils like `getWidth`/`getHeight` to integrate with a `visualViewport` api